client/src/styles/new-song-form.css

```diff
@@ -0,0 +1,71 @@
+.new-song-form {
+    width: 50%;
+    margin: 2% auto 0;
+    background: white;
+    border: 2px solid #e6e6e6;
+    border-radius: 4px;
+}
+
+.song-link-input {
+    margin-top: 3%;
+    height: 20px;
+    width: 80%;
+    background: #fafafa;
+}
+
+.select-format {
+    width: 80%;
+    margin-top: 3%;
+    height: 25px;
+    border: 1px solid #e6e6e6;
+    border-radius: 4px;
+    font-size: 18px;
+    transition: background-color 3s ease;
+    background: #fafafa;
+}
+
+.add-song-button {
+    font-size: 18px;
+    margin-top: 4%;
+    margin-bottom: 2%;
+}
+
+.song-form-button:hover {
+    background: #222222;
+}
+
+.loader {
+    margin: 2% auto;
+    border: 16px solid #f3f3f3; /* Light grey */
+    border-top: 16px solid #222222; /* Blue */
```

Comment:
If this color is **exactly** *blue* you can just write `blue` as in:
```css
border-top: 16px solid blue

Reply:
lol